Frontend/src/store/search/actions.ts

```diff
@@ -41,3 +44,21 @@ export const setSearchLoading = (isLoading: boolean): SearchActionTypes => ({
   type: SET_SEARCH_LOADING,
   payload: isLoading,
 })
+
+export const filter = (): SearchActionTypes => ({
+  type: FILTER,
+})
+
+export const setFilters = (filters: FiltersValues, instantSearch: boolean): SearchActionTypes => ({
```

Comment:
setFilters a resetFilters jsou zatím jen nějaká příprava? nikde to nemůžu najít, že by se to volalo

Reply:
@boriscirok je to tak, ony se ty filtry nejspíše rozšíří tak jsem tam pár věcí připravil..